packages/eslint-config/README.md

```diff
@@ -55,6 +55,43 @@ Run:
 - `npm run lint:eslint` to check if any file is not correctly formatted
 - `npm run format:eslint` to auto format your files
 
+## Running with Prettier
+
+If you also are using Prettier, in order for your ESLint to play nicely with Prettier, you should also include eslint-config-prettier.
+This way, the Prettier formatting will take into account your linting rules, so you don't need to add your own overrides for Prettier if it clashes with your linting rules.
+
```

Comment:
I would like to add here:

```
If you add Prettier overrides that clash with your linting configuration, the linting will take precedence and the Prettier override will not apply. So take this into account as well.
```
But I'm not sure if that's actually correct, so if someone could verify :) I will add it